src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border--color;
+  border-radius: 5px;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image-container-img {
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__product-info {
+    margin-top: 40px;
+    margin-bottom: 16px;
+  }
+
+  &__product-info-text {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main--accent;
+  }
+
+  &__product-info-code {
+    margin-top: 4px;
+    color: $secondary--accent;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__product-review {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__product-review-number-of-reviews {
+    margin: 0;
+    color: $main--accent;
+    font-size: 10px;
+  }
+
+  &__price-container {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-container-text {
+    margin: 0;
+    font-size: 12px;
+    color: $secondary--accent;
+    line-height: 18px;
+  }
+
+  &__price-container-number {
+    margin: 0;
+    color: $main--accent;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-btn {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    color: $main--white;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $blue--accent;
+    border-radius: 5px;
+  }
+
+  &:hover {
+    border: 1px solid $blue--accent;
+    color: #00acdc;
+    background-color: $main--white;
+  }
```

Ховер ефект стосується кнопки, а не картки
```suggestion
  &__buy-btn {
    height: 40px;
    display: flex;
    align-items: center;
    justify-content: center;
    line-height: 16px;
    font-size: 14px;
    font-weight: 700;
    color: $main--white;
    text-decoration: none;
    text-transform: uppercase;
    background-color: $blue--accent;
    border-radius: 5px;

	  &:hover 
	    border: 1px solid $blue--accent;
	    color: #00acdc;
	    background-color: $main--white;
	  }
   }
```